### PR TITLE
Better layout for entry list

### PR DESCRIPTION
**Pull request by Atmos4**
- better layout for entry list:  
image
- switch logout and user feedback in navbar:
image

### CODE REVIEW

File: engine/core/Component.php [hunk 1]
```diff
@@ -5,21 +5,21 @@ class Component
     public static array $_props = [];
     static function render(string $location, array $props = []): string
     {
-        self::$_props = $props;
         ob_start();
+        self::$_props[ob_get_level()] = $props;
```
Comment (Atmos4) [inline engine/core/Component.php:9]: 2-dimensional array allows for infinite component depth

I might build a more robust templating system in the future, taking inspiration from https://platesphp.com/

**Comment by Mathieu414** [comment]
Also don't forget to remove the "Inscrits" button inside the race accordions

File: app/pages/user_feedback_submit.php [hunk 2]
```diff
@@ -10,7 +10,7 @@
     $new_user_feedback->description = $description->value;
     em()->persist($new_user_feedback);
     em()->flush();
-    Toast::success("Merci pour votre retour !");
```
[the remark this answers is not in the document]
Comment (Atmos4) [reply app/pages/user_feedback_submit.php:13]: We don't have super good consistency throughout the project, but I think using informal here is ok. Also it makes it consistent with the page

File: app/pages/events/entry_list/entry_list_tab_race.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<?php
+$raceId = Component::prop("race_id");
+$raceEntries = RaceService::getRaceEntries($raceId);
+?>
+<?php if (!$raceEntries): ?>
+    <p>Pas d'inscrits sur cette course</p>
```
[the remark this answers is not in the document]
Comment (Atmos4) [reply app/pages/events/entry_list/entry_list_tab_race.php]: Done with inline styles because I'm lazy :D

File: app/pages/events/event_view.php [hunk 4]
```diff
@@ -128,70 +125,64 @@
...
-                                    <?= format_date($race->date) ?>
-                                </li>
-                                <li><span class="fa-li"><i class="fa fa-location-dot"></i></span>
-                                    <?= $race->place ?>
+        <h4>Courses :
+            <a href="#" title="Afficher tous les détails" onclick="
+                    htmx.findAll(htmx.closest(this, 'article'),'details').forEach(e => {e.open=!e.open});
+                    return false
+                ">
+                <i class="fa fa-up-down"></i>
```
Comment (Mathieu414) [inline app/pages/events/event_view.php:133]: Great idea, love it ! I would rather use the `angles-down` icon from fa though, maybe like that :

Capture d'écran 2023-12-16 195042

Comment (Mathieu414) [reply app/pages/events/event_view.php:133]: Or like that : 

Capture d'écran 2023-12-16 195416

Comment (Mathieu414) [reply app/pages/events/event_view.php:133]: Okay for the current icon 🫡

**Review by Mathieu414 — DISMISSED** [review]

File: app/pages/events/event_view.php [hunk 5]
```diff
@@ -128,70 +125,64 @@
...
-                        <div class="col-sm-12 col-md-6">
-                            <ul class="fa-ul">
-                                <li><span class="fa-li"><i class="fa fa-calendar"></i></span>
-                                    <?= format_date($race->date) ?>
-                                </li>
-                                <li><span class="fa-li"><i class="fa fa-location-dot"></i></span>
-                                    <?= $race->place ?>
+        <h4>Courses :
+            <a href="#" class="secondary" title="Afficher tous les détails" onclick="
+                    htmx.findAll(htmx.closest(this, 'article'),'details').forEach(e => {e.open=true});
```
Comment (Mathieu414) [inline app/pages/events/event_view.php:130]:
```suggestion
                    htmx.findAll(htmx.closest(this, 'article'),'details').forEach(e => {e.open=!e.open});
```

**Review by Mathieu414 — APPROVED** [review]

**Pull request merged**